workloads/librispeech/README.md

```diff
@@ -8,22 +8,38 @@ and evaluate it on Librispeech test clean dataset. We get Character Error Rate
 
 ## 2. Download and preprocess dataset
 
-```
+```bash
 mkdir data_dir
+# Make script executable
+chmod +x download_data.sh
 cd data_dir
 
-download_data.sh
+# Download and unzip LibriSpeech dataset
+../download_data.sh
+
+# [Optional] Delete the tar.gz files to save disk space
+# rm *.tar.gz
 
 cd ..
 mkdir work_dir
 cd work_dir
 
-python prepare_data.py data_dir/LibriSpeech
+# Run preprocessing script
+python ../prepare_data.py ../data_dir/LibriSpeech/
```

Comment:
are the `../` necessary? aren't those scripts in this dir?

Reply:
same with above `../download_data.sh`

Reply:
The `../` is used because both the `download_data.sh` and `prepare_data.py` is coded to store the data in the dir from which it runs. 
`download.sh` is run from `data_dir` and that's where the data get downloaded and unzipped.

I can update both scripts take path argument and solve this. 

Let me know your thoughts on this.